packages/gatsby-source-graphql/src/gatsby-node.js

```diff
@@ -7,7 +7,7 @@ const {
   RenameTypes,
 } = require(`graphql-tools`)
 const { createHttpLink } = require(`apollo-link-http`)
-const fetch = require(`node-fetch`)
```

Comment:
This variable name was used in line 53 as a shorthand property.

Could you please try to add a test that catches this regression? This would help as clearly that path is currently not covered.

And then fix the shorthand by `fetch: nodeFetch` :)

Reply:
Line 28 defaults `fetch` to `nodeFetch` in the function scope. We are specifically changing the fetch that we pass to `createHttpLink` on line 53. 

I wrote tests to check when you pass and don't pass the `fetch` in as an option.

Reply:
Oh yeah, wow. Sorry about that :)